### DISCUSSION TITLE
2024 - Sugggested host CPU for Pentium II 266Mhz and PII 400Mhz

### DISCUSSION THREAD

**Discussion by Mal752** [86Box/86Box#4943] [Q&A] [answered]
Hi, with current gen processors - Core 7 Ultra and Ryzen AI 9 HX 375 available, will these be good CPUs for running a Pentium II 266MHz smoothly without any hiccups or slowdown? Any other older Ryzen 7 Pro or Core i7 can do this?

And are there any CPUs that can run a PII 400Mhz based system without any slowdown ever (for Windows 98)?  

(My desktop Ryzen 9 5900X can't run PII 400Mhz smoothly. Too many slowdowns to run games properly. Even a 266MHz has some hiccups here and there, enough to spoil a smooth experience.)

Thank you.

P.S. I'm in the process of getting a new laptop and would like to know which can run the above smoothly.  (My current laptop is a HP Pavilion 15 with Core i7 9750H. I can run a Pentium 133MHz perfectly and a Pentium 166Mhz without too much issues.)

**Comment by kjliew** [comment]
A historic [quote of evidence](https://www.reddit.com/r/86box/comments/1gix66u/comment/lvj1572/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) from super-talented 86Box lead developer @OBattler 
>You're the only one insisting you need to emulate a high-speed Pentium II to play those games decently, because you're the only one insisting that the only decent way to play such games is to play them at modern resolutions and framerates.

Expect further discussions to be abruptly _PURGED_ out of censorship as a result from the heat of humiliation. Look elsewhere if you ever need such CPUs to play your games. OR come back with the same question 15 years later OR if mankind achieved quantum computing break-through.

OR  just run those games directly on your host. All you have to do is as simple as possibly download a free patch.

**Reply by jriwanek** [reply]
Or I suppose charge what was it? $90 for builds of your stuff, don't you have anything better to do with your life mate?

**Reply by kjliew** [reply]
Oh GOOD mate, didn't you step on the line of canvassing 86Box user for that _EVIL_ project?

Let the choice be OPEN for being professional. A $90 for Pentium III 800MHz with GeForce/Radeon DX9-class performance on existing Core i7 9750H or a new laptop for marginal improvement with the same old, crappy voodoos?

OR how could you be so sure that OP doesn't possess the skills to build that _EVIL_ project from source or make good connections with someone who can do it for him?

**Reply by jriwanek** [reply]
Hey, considering as you say people can just use patches for free, and native, why charge for your builds at all? We don't, people choose whether they want to use our stuff or not, we're not trying to trap people who don't know how to compile into paying, nor are we showing up on your issue tracker acting like you do, as to canvassing? What on earth are you on about? Only person I've seen canvassing is you, so, kindly take your nonsense and blather elsewhere.

**Reply by kjliew** [reply]
I leave a [link](https://www.reddit.com/r/windowsxp/comments/1caccam/has_anyone_used_qemu3dfx_with_windows_xp_before/) as the **_evidence of proof_** that people can just use patches for _FREE_ with sufficient due diligence and to sinisterly canvass 86Box users who are clueless about modern virtualization. Try it out if you may, and let's see if someone can learn something and submit a PR for 86Box to "virtualize" for games.

Didn't the mob of 86Box rigorously show up in comments on Linus "comedy" Tech Tips premiere of PCem? Wouldn't you consider that also an act  of "shameless" canvassing, too?

**Reply by OBattler** [reply]
@kjliew Did you I certainly didn't appear in the comments section of that video, if any of the rest of my team did, well, I'd like to see what they actually said. There may have been people actually talking about 86Box so they answered the questions. That would be different from just barging in and tell people "PCem sucks, use 86Box instead". We, in fact, used to do that in 2015, we were told to cut the crap, and we did and learned the lesson.

**Reply by OBattler** [reply]
> A historic [quote of evidence](https://www.reddit.com/r/86box/comments/1gix66u/comment/lvj1572/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button) from super-talented 86Box lead developer @OBattler
> 
> > You're the only one insisting you need to emulate a high-speed Pentium II to play those games decently, because you're the only one insisting that the only decent way to play such games is to play them at modern resolutions and framerates.
> 
> Expect further discussions to be abruptly _PURGED_ out of censorship as a result from the heat of humiliation. Look elsewhere if you ever need such CPUs to play your games. OR come back with the same question 15 years later OR if mankind achieved quantum computing break-through.

While I did ban you from the subreddit, none of your posts, other than the three canvassing ones, have been purged.

> OR just run those games directly on your host. All you have to do is as simple as possibly download a free patch.

Which is something **I** have been saying, so now you're literally using that which **I** suggested as an alternative to your $90 per platform per binary project, against me.

**Reply by DDXofficial** [reply]
> Expect further discussions to be abruptly _PURGED_ out of censorship as a result from the heat of humiliation. Look elsewhere if you ever need such CPUs to play your games. OR come back with the same question 15 years later OR if mankind achieved quantum computing break-through.

Hi kjliew! As you remember, I was the first one (yes, I am [u/TV-MA-L-S-V](https://www.reddit.com/user/TV-MA-L-S-V/) on Reddit) to remove you from the subreddit not for the sake of "censorship" as you erroneously claim, but purely because you are utterly incapable of arguing and providing any form of criticism in good faith _and_ feel the need to advertise your grift of a QEMU fork ($90, seriously?) while, at the same time, harassing novice users when they sought for any kind of assistance over there. This, unfortunately, is not the atmosphere I wish to foster in the community, and is why you were removed _once again_, and permanently this time, from the subreddit.

Now, I will admit that choosing to so unprofessionally ban you with the message "[insert expletive here], your toxic attitude isn't welcome here" - or something of that nature as this happened months ago in the heat of the moment - is definitely not something I would have ever done in my 5+ years of community management, and I apologize for that. However, your inflammatory messages, for which I have never seen a reason, have been constantly reported to me for showing an inherent lack of respect towards other users and whatever hardware they have either owned for years due to circumstance or purchased recently, and neither I nor OBattler were going to stand for that.

Time and time again, your aggressive and negative attitude in the subreddit has caused me and everyone managing this project to lose any form of respect we've had for you despite your skills. Even regular users took it upon themselves to call you out on your grift before we took action. If this doesn't show that you have irreversibly caused damage to your reputation not only in this project's community, but also in other retro computing emulation circles as well, then I fear I don't know what else will.

Now sure, I'll take your jab at the Computer Engineering degree I'm currently pursuing on the chin. I don't doubt you are considerably more skilled and knowledgeable at computer architecture design, low-level programming, and other related disciplines that I currently am. But why should I, or anyone else for that matter, give you or your QEMU fork the time of day when all I have ever known about you is your brazen, prideful attitude paired with the constant desire to put down anyone who even so much as dared to use our emulator for the purpose they choose?

I will take this moment to remind you that each tool serves its distinct purpose despite performing similar tasks. Sure, one can run an older version of Windows inside VirtualBox/VMware/QEMU and 86Box/PCem, but both software groups are vastly different in that one group deals with hardware virtualization while the other emulates an entire computer in software. If one chooses one over the other, it is their choice and it is not your place to pontificate to them about why YOUR method is better.

I pray you take this to heart while you can, because your days of interacting with this community are soon to be numbered.

**Comment by Mal752** [comment]
Whoa... so such things happened before, huh? Never knew about it since I never visited Discussions section before. Only posted bugs in the Issues section.

I was just thinking about it since the PII is included in the emulated machines. So, my guess is, no current CPUs can run a PII system in 86Box smoothly?

Anyway, What do you think will be a good CPU for emulating a Pentium 200 MMX smoothly, then? 

There are some problematic Win9x based games, mostly those that use DirectDraw that has problems in current OS.  

I'm using Linux in my systems, and WINE does a better job for these DirectDraw based games compared to Windows, but I have noticed some that do have difficulties.  Robin Hood The Legend of Sherwood, Revenant, etc.

Anyway, let's drop the discussion of the capability of 86Box to run PII system smoothly for now, then. 

During the era of Pentium, I had a Pentium 133 system and had a blast with it with the games that came on CD-ROM back then and some still coming in floppies. And I am happy to see that in 86Box, the Pentium 133 "machine" works beautifully with my current desktop and laptop. I'm glad I am able to relive those days anywhere anytime now with my laptop.

**Reply by jriwanek** [reply, marked as answer]
> I was just thinking about it since the PII is included in the emulated machines. So, my guess is, no current CPUs can run a PII system in 86Box smoothly?

It really depends on what you're doing, and what speed of PII really, My CPU (I7-12900K) can manage the High 300's pretty stably.

> Anyway, What do you think will be a good CPU for emulating a Pentium 200 MMX smoothly, then?

I'm told a M1 mac can do 233 alright, as can a 5800X, A R5 5500U, anything which can score in the range of ~3400 on cpu benchmark should be pretty suitable really, so for intel around an i5-12600H or up.

Useful info: If you're getting drops, you can try switching in the settings from nuked to ymfm, playing with the HDD speed emulation, and switching dynarec (The ODR or NDR builds).

**Reply by kjliew** [reply]
Any modern CPUs including the 8W TDP Intel N100 can easily emulate in CPU performance exceeding Pentium II 400MHz for software in the know-how to "virtualize".

>I'm using Linux in my systems

And you're clueless about QEMU on Linux all these years for its superb PC system emulation with KVM? How unfortunate! Revenant even in software 3D plays much better in QEMU KVM than 86Box emulating Pentium 200 MMX with the fantastic Voodoos in Direct3D.

**Reply by Mal752** [reply]
I'm not familiar with QEMU KVM for emulating a Windows9x based system. Can you help provide a link to the detailed guide? Thanks. 

Edit: It's okay, I'll search myself. But thanks for letting me know about this possibility. I will focus on this route and see.

And no. Not going to touch the ripoff "3dfx fork". No thanks.

**Reply by OBattler** [reply]
@Mal752 He's talking about his QEMU-3Dfx fork which costs $90 per platform for binaries, and the support provided is literally "Google it". He's been harassing this and other projects for ages now.

**Reply by Mal752** [reply]
I see. That is very bad. Quite shameful, I might add.

Thanks for maintaining 86Box, @OBattler and the team. I am able to relive all my nostalgia right from the day I got my XT.

**Reply by Alexander207** [reply]
>anything which can score in the range of ~3400 on cpu benchmark

What CPU benchmark if I may ask?
